### ISSUE TITLE
Consider adding some accessibility support to (over) bootstrap

### ISSUE THREAD

**Issue by nadavkav** [gjb2048/moodle-theme_essential#507]
I was reading about bootstrap accessibility, and thought we could add some on the Essential side, what do you think?

http://www.sitepoint.com/making-bootstrap-accessible/

**Comment by gjb2048** [comment]
Hi Nadav,

Ta.  I've heard and used the linked site http://wave.webaim.org/ for over a year.  The issue is not desire its time.  If you would like to suggest / make changes to settings defaults for colours / font sizes etc. then please do. :)

Gareth

**Comment by nadavkav** [comment]
Thanks! I am only starting to research into this area since the Israeli government passed an accessibility law, and all Academic institutes and other education related organizations are all asking for themes that are supporting accessibility standards. And so, I was wondering if you made any specific changes in the theme relating to that?

**Comment by gjb2048** [comment]
Not that I recall.  Please look back on issue list.

**Comment by nadavkav** [comment]
I was looking for I posted and did not found any. maybe my english search was not good enough.
Please enlighten me if you know of anything :-)

**Comment by nadavkav** [comment]
BTW, the wave.webaim.org tool is so useful! thanks.

**Comment by gjb2048** [comment]
In Essential I don't think so.

**Comment by gjb2048** [comment]
Hi Nadav,

This is a few months old now.  What is happening with it?

G

**Comment by nadavkav** [comment]
A fellow developer, from the Israeli Open University, just finished a lot of accessibility work on the Elegance theme, last week. And he invited me to look around the code and see what can be shared and used in Essential. So I will probably pay him a visit next week or so, as we are in the middle of our annual holiday season, and nobody is working :-) 

I will keep you posted on this.

**Comment by gjb2048** [comment]
Thanks Nadav :)

**Comment by gjb2048** [comment]
What is the progress of this?

**Comment by nadavkav** [comment]
I got the code, two months ago. started to dig in to it and make sense out of it, as it was from an old old version of the theme and much have changed, but did not finished. plus, I got soooo busy with other previous obligations that I paused the entire process.
Tomorrow, I am having a small Moodle gathering of all the academic institutes over here, and I am planning on presenting the state of the project and hoping to gather enough support and funding to finish it.

So in short... It is "in progress" but not yet ready for a pull request.

A little preview of things to come...
I am attaching an animated gif showing the accessibility menu opens up when I click the TAB key and show the main course topics which allow the user to quickly navigate to them (skipping all the garbage element on the way). It's in Hebrew. but still, very clear. I hope.
![animated](https://cloud.githubusercontent.com/assets/231168/13204859/f8fc2bdc-d8e2-11e5-9ee8-05cf147a29e2.gif)

**Comment by gjb2048** [comment]
Thanks Nadav.  Has it been tested with screen readers such as JAWS?

**Comment by nadavkav** [comment]
Yes (I think so. or a similar software. I will check tomorrow). successfully.
We also tested it with actual people (blind students) that use special speaking browsers, daily.
Not just following the WCAG 2 specs :-)

**Comment by gjb2048** [comment]
Thanks Nadav.  It's that sort of good testing that is so difficult to get.  Such as with the Grid format.

**Comment by gjb2048** [comment]
Hi Nadav,

Is this still an issue?

Kind regards,

Gareth

**Comment by nadavkav** [comment]
I am getting another Accessibility auditing to some of the Moodle systems I manage, in a few weeks. Two of them have Essential installed. so I will PR any Ally issues we fix. but currently, I do not have any special code fix I can share.

**Comment by gjb2048** [comment]
Thanks for getting back to me Nadav.

**Comment by gjb2048** [comment]
So does this issue need to close in favour of just raising new ones as required?

**Comment by nadavkav** [comment]
Yes
